.pre-commit-config.yaml

```diff
@@ -59,3 +60,8 @@ repos:
       - id: pylint
         args: [--disable=E0401]
         exclude: db_migration|.devcontainer
+  - repo: https://github.com/pycqa/isort
```

Comment:
As isort modifies our code, we probably should run it either before black, or right after black.

Reply:
In pre-commit that doesn't matter, as when there are changes made by a hook they have to be first manually staged to be considered for the other hooks. But to have it logically consistent, I will change it.

Reply:
done